.codesandbox/ci.json

```diff
@@ -2,7 +2,8 @@
   "sandboxes": [
     "vanilla",
     "vanilla-ts",
-    "github/reduxjs/rtk-github-issues-example"
+    "github/reduxjs/rtk-github-issues-example",
+    "github/reduxjs/toolkit/examples/query/react/optimistic-updates"
```

Comment:
```suggestion
```
I don't think we'd need all the examples for the docs built this way, but I could be missing something.

Reply:
Can you specify?